### PR TITLE
Open file implementation

### PR DESCRIPTION
**Pull request by jooohhn**
- Falcon can open files after being packaged. Not sure how to do from dev.
- Only works for macOS

Steps: 

1. `yarn package`
2. Move to Falcon Applications from `.dmg` 
3. Try opening when app is closed and when app is open

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,6 +42,16 @@
       "main.js.map",
       "package.json"
     ],
+    "fileAssociations": [
+      {
+      "ext": "sqlite",
```
Comment (amilajack) [inline package.json]: Indentation is off here

**Comment by amilajack** [comment]
This fails on windows. [Here's the stactrace](https://ci.appveyor.com/project/amilajack/falcon/build/37#L156) on appveyor. Lets fix this and then merge

**Comment by jooohhn** [comment]
Closes #41

**Comment by amilajack** [comment]
Why was this closed?

**Pull request merged**